NadavAI/visualization/visualization.cpp

```diff
@@ -22,18 +21,15 @@ Agent::Agent(const Entity &entity) {
 
 Direction Agent::radiansToDirection(Radian angle) {
     auto degrees = angle.toDegrees();
-    if ((degrees >= 0 && degrees <= 45) || (degrees >= 315 && degrees <= 360)) {
+    if (degrees >= 225 && degrees <= 315) {
```

Comment:
optionally don't convert to degrees and use `angle.between(from, to)` which has tests

Reply:
fixed gracias maestro